src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperOutput.cs

```diff
@@ -96,7 +83,7 @@ public bool ContentSet
         {
```

Comment:
Now since we have the chance, modify this property to be: `public bool ContentSet => Content.IsContentModified()`.